src/main/java/net/sf/jabref/logic/pdf/FileAnnotationCache.java

```diff
@@ -0,0 +1,43 @@
+package net.sf.jabref.logic.pdf;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import net.sf.jabref.model.database.BibDatabaseContext;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.pdf.FileAnnotation;
+
+import org.apache.commons.collections4.map.LRUMap;
```

Comment:
Could we use a class from google  guava instead? Because of this LRUMap class, the whole commons collections library has to be included.